inputs/autocomplete.vue

```diff
@@ -90,22 +105,37 @@
         }
       }
     },
-    mounted() {
-      const listName = this.args.list,
-        lists = this.$store.state.lists,
-        items = _.get(lists, `${listName}.items`);
+    methods: {
+      fetchListItems() {
+        const listName = this.args.list,
+          lists = this.$store.state.lists,
+          items = _.get(lists, `${listName}.items`);
+        let promise;
+
+        if (items) {
+          promise = Promise.resolve(items);
+        } else {
+          promise = this.$store.dispatch('getList', listName).then(() => _.get(lists, `${listName}.items`));
+        }
 
-      let promise;
+        return promise.then((listItems) => {
+          this.listItems = _.map(listItems, (item) => _.isObject(item) ? item.text : item);
+        });
+      },
+      removeFromList(item) {
+        const listName  = this.args.list,
+          unselect = this.unselect;
 
-      if (items) {
-        promise = Promise.resolve(items);
-      } else {
-        promise = this.$store.dispatch('getList', listName).then(() => _.get(lists, `${listName}.items`));
-      }
+        unselect();
```

Comment:
I'm not sure why this needs to be a variable. It doesn't look like it's reused anywhere?

Reply:
Made it a variable for a previous use case that's not applicable anymore. changing!